### PR TITLE
Feat: Added Navbar Component

### PR DESCRIPTION
**Pull request by shuntzhere**
Screenshot (25)

### CODE REVIEW

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.navigation-container > nav {
```
Comment (krishanu2911) [inline components/navbar/navbar.css:1]: Just suggestion in class naming it could be .nav-container as below you declared .nav-logo

**Review by krishanu2911 — COMMENTED** [review]
Added a suggestion Else seems good to go well-written code.

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
+.card {
+  height: 450px;
+  width: 350px;
+  background-color: white;
```
Comment (shubhamkumar648) [inline components/card/card.css:4]: use variable for color

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+.card-text {
+  display: inline-block;
+  margin: 0.2rem auto;
+  font-size: 1rem;
+  color: #474747;
+}
+
+.card-btn > p {
+  color: #8d8d8d;
```
Comment (shubhamkumar648) [inline components/card/card.css:31]: same here

**Review by shubhamkumar648 — COMMENTED** [review]
consider my comment as a cahnges , other looks good

File: components/card/card.css (hunk 2 above)
Comment (shuntzhere) [reply components/card/card.css:4]: Yaah will change that thanks.

**Review by krishanu2911 — APPROVED** [review]

**Review by shubhamkumar648 — APPROVED** [review]
Remaining look good

**Pull request merged**